### ISSUE TITLE
What is cross entropy loss

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#72]
Cross-entropy loss, or log loss, measures the performance of a classification model whose output is a probability value between 0 and 1. Cross-entropy loss increases as the predicted probability diverges from the actual label. So predicting a probability of .012 when the actual observation label is 1 would be bad and result in a high loss value. A perfect model would have a log loss of 0.

![](https://ml-cheatsheet.readthedocs.io/en/latest/_images/cross_entropy.png)

In binary classification, where the number of classes 𝑀 equals 2, cross-entropy can be calculated as:

−(𝑦log(𝑝)+(1−𝑦)log(1−𝑝))

If 𝑀>2 (i.e. multiclass classification), we calculate a separate loss for each class label per observation and sum the result.

−∑𝑐=1𝑀𝑦𝑜,𝑐log(𝑝𝑜,𝑐)
